js/ui/codeView.js

```diff
@@ -643,8 +644,8 @@ var CodingSession = new Lang.Class({
         if (!this.builder) {
             // get the manifest of the application
             // return early before we setup anything
-            let appManifest = _getAppManifest(this.app.meta_window.get_flatpak_id());
-            if (!appManifest) {
+            this.appManifest = _getAppManifest(this.app.meta_window.get_flatpak_id());
```

Comment:
There should be no reason for this to be "public". You can use `this._appManifest`